### PR TITLE
CI: Add required signoffs

### PR DESCRIPTION
**Pull request by merlin-northern**
Change-noadded-Log:none
Signed-off-by: Peter Grzybowski <peter@northern.tech>

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 include:
   - project: 'Northern.tech/Mender/mendertesting'
     file: '.gitlab-ci-github-status-updates.yml'
+  - project: 'Northern.tech/Mender/mendertesting'
+    file: '.gitlab-ci-check-commits.yml'
```
Comment (lluiscampos) [inline .gitlab-ci.yml]: We don't need changelogs as we don't release this from `release_tool.` I would align this repo with other internal libraries like `go-lib-micro`.

**Comment by lluiscampos** [comment]
> Change-noadded-Log:none

Testing? :)

**Comment by merlin-northern** [comment]
> > Change-noadded-Log:none
> 
> Testing? :)

of course!

File: .gitlab-ci.yml (hunk 1 above)
Comment (lluiscampos) [reply .gitlab-ci.yml]: After we reach an agreement, and once you are on it, please add the template also to `go-lib-micro` (it is missing).

Comment (merlin-northern) [reply .gitlab-ci.yml]: @oleorhagen your opinions? (you requested change logs :))
I personally kind of like the change logs, but I do not mind keeping only signoffs -- no strong opinions here.

Comment (merlin-northern) [reply .gitlab-ci.yml]: we decided that we require only signoffs.

**Review by oleorhagen — APPROVED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (merlin-northern) [reply .gitlab-ci.yml]: > After we reach an agreement, and once you are on it, please add the template also to `go-lib-micro` (it is missing).

[sure thing](https://github.com/mendersoftware/go-lib-micro/pull/76)!

**Pull request merged**